src/client.js

```diff
@@ -31,6 +31,7 @@ export const clientListen = function () {
                         xhr.httpResponseCode = this.status;
                         xhr.responseData = this.response;
                         xhr.status = (this.status === 200 ? 'success' : 'error');
+                        xhr.headers = this.getAllResponseHeaders();
```

Comment:
Thanks for your contribution ! 

I suggest the new field to be called `responseHeaders`, in case we ever add `requestHeaders` ....